lib/identity/parsers/gc_ids_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Identity
+  module Parsers
+    module GCIdsHelper
+      # @param edipi[String] An string representing the edipi id
+      # @return [String] A string of 10 numerical digits representing the sanitized id
+      def sanitize_edipi(edipi)
+        return unless edipi.present? && edipi.is_a?(String)
```

Comment:
If we want to ensure that edipi is present and a String should we raise an error rather than just return nil? That way it never actually will return without the correct argument?

Reply:
I think that's not a bad point, I think for now I'm going to leave it, and the reason is that the only place that invokes this function is `lib/mpi/responses/profile_parser.rb`, and it very explicitly wants `nil` rather than an error when it calls `sanitize_edipi` and `sanitize_id`.
Overall we might want to consider just rolling these sanitize functions into into the parsed GCIds hash itself, and I think that could be a good way to address this feedback.

Reply:
Although a question I do have:
- `sanitize_edipi` is used in the profile_parser only
- `sanitize_id` is used in add_parser, profile_parser, and ssoe
Does it seem like these aren't actually parser functions, and would be better served as a formatter or something? They're pretty small functions so might not make sense to make a mountain out of a molehill here, but both of these functions are in a strange spot currently...